keps/sig-auth/20190607-certificates-api.md

```diff
@@ -292,8 +291,29 @@ type CertificateSigningRequestStatus struct {
   // Conditions applied to the request, such as approval or denial.
   Conditions []CertificateSigningRequestCondition
 
-  // If request was approved, the controller will place the issued
-  // certificate here.
+  // Certificate is populated by the signer with the issued certificate in PEM format.
+  // In JSON and YAML output, this entire field is base64-encoded, so it consists of:
+  // base64(
+  // -----BEGIN CERTIFICATE-----
+  // MII...Pb7Yu/E=
+  // -----END CERTIFICATE-----
+  // optional intermediate certificate blocks
+  // -----BEGIN CERTIFICATE-----
+  // MII...MGKB
+  // -----END CERTIFICATE-----
+  // -----BEGIN CERTIFICATE-----
+  // MII...AY1M
+  // -----END CERTIFICATE-----
+  // )
+  //
+  // In v1beta1, this field is unvalidated.
+  //
+  // In v1, this field is validated to contain one or more PEM blocks.
+  // All PEM blocks must have the "CERTIFICATE" label, contain no headers, and the encoded data
+  // must be a BER-encoded ASN.1 Certificate structure as described in Section 4 of RFC5280.
```

Comment:
(taken from https://tools.ietf.org/html/rfc7468#section-5.1)

Reply:
Add link to the markdown?

Reply:
probably not in API doc